src/js/embed.js

```diff
@@ -66,50 +66,65 @@ window.init = function init(parentEl, config) {
 
     setupVisibilityMonitoring();
 
-    reqwest({
-        url: 'https://interactive.guim.co.uk/docsdata/1zsqQf4mq8fsAkZAXnoSCNpap2hykFDA3Cm3HaI9qe8k.json',
-        type: 'json',
-        crossOrigin: false,
-        success: (res) => {
-            const params = getQueryParams();
-            const {sheet, id, format = 'flat'} = params;
-            const rows = res && res.sheets && res.sheets[sheet];
-            const template = formats[format] && formats[format].template;
-            const templateFn = dot.template(template);
-            const trackingCode = 'brexit__' + sheet + '__' + id;
-            let data;
-            let row;
-
-            if (!rows || !rows.length) {
-                console.log('bad JSON response');
-                console.log(res);
-
-                return;
-            }
-            rows.forEach((r) => {
-                if (r.id === id) {
-                    row = r;
-                }
+    async.parallel({
+        spreadsheetReq: function(callback){
+            reqwest({
+                url: 'https://interactive.guim.co.uk/docsdata/1zsqQf4mq8fsAkZAXnoSCNpap2hykFDA3Cm3HaI9qe8k.json',
+                type: 'json',
+                crossOrigin: false,
+                success: function(res) { callback(null, res); }
             });
-            if (!row) {
-                console.log('row with id ' + id + ' not found');
-
-                return;
-            }
-            if (!template) {
-                console.log('format ' + format + ' is not valid');
-
-                return;
+        },
+        tailorReq: function(callback){
+            reqwest({
+                url: 'https://members-data-api.theguardian.com/tailor/me?tag=politics/eu-referendum',
+                type: 'json',
+                crossOrigin: false,
+                success: function(res) { callback(null, res); }
+            });
+        }
+    },
+    function(err, results) {
+        // results is now equals to: {one: 1, two: 2}
+
+        var $__4;
```

Comment:
Yes! Gah, please ignore all this - I accidentally edited the 'build' version of embed.js and then did some bad copy-and-paste into the 'src' file when I realised my mistake.